reports-scheduler/src/main/java/com/amazon/opendistroforelasticsearch/reportsscheduler/job/parameter/JobParameter.java

```diff
@@ -0,0 +1,112 @@
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.reportsscheduler.job.parameter;
+
+import java.io.IOException;
+import java.time.Instant;
+
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.ScheduledJobParameter;
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.schedule.Schedule;
+
+/**
+ * A report scheduled job parameter.
+ *
+ * <p>It adds an additional "reportDefinition" field to {@link ScheduledJobParameter}, which stores
+ * the index the job runner will watch.
+ */
+public class JobParameter implements ScheduledJobParameter {
+
+  private final String jobName;
+  private final Instant enabledTime;
+  private final String reportDefinitionId;
+  private final boolean isEnabled;
+  private final Schedule schedule;
+  private final Instant lastUpdateTime;
+  private Long lockDurationSeconds;
+
+  public JobParameter(
+      String jobName,
+      Instant enabledTime,
+      String reportDefinitionId,
+      boolean isEnabled,
+      Schedule schedule,
+      Instant lastUpdateTime) {
+    this.jobName = jobName;
+    this.enabledTime = enabledTime;
+    this.reportDefinitionId = reportDefinitionId;
+    this.isEnabled = isEnabled;
+    this.schedule = schedule;
+    this.lastUpdateTime = lastUpdateTime;
+  }
+
+  @Override
+  public String getName() {
+    return this.jobName;
+  }
+
+  @Override
+  public Instant getLastUpdateTime() {
+    return this.lastUpdateTime;
+  }
+
+  @Override
+  public Instant getEnabledTime() {
+    return this.enabledTime;
+  }
+
+  @Override
+  public Schedule getSchedule() {
+    return this.schedule;
+  }
+
+  @Override
+  public boolean isEnabled() {
+    return this.isEnabled;
+  }
+
+  @Override
+  public Long getLockDurationSeconds() {
+    return this.lockDurationSeconds;
+  }
+
+  public String getReportDefinitionId() {
+    return this.reportDefinitionId;
+  }
+
+  public void setLockDurationSeconds(Long lockDurationSeconds) {
+    this.lockDurationSeconds = lockDurationSeconds;
+  }
+
+  @Override
+  public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+    builder.startObject();
+    builder
+        .field(JobConstant.NAME_FIELD, this.jobName)
+        .field(JobConstant.ENABLED_FILED, this.isEnabled)
+        .field(JobConstant.SCHEDULE_FIELD, this.schedule)
+        .field(JobConstant.REPORT_DEFINITION_ID, this.reportDefinitionId);
+    if (this.enabledTime != null) {
+      builder.timeField(
+          JobConstant.ENABLED_TIME_FILED,
```

Comment:
np: consistent indentation

Reply:
I am using google-java-format to format the code, the same as Elasticsearch source code and ODFE sql. It's using 2-space indentation. It seems like when you start a new line but in the same statement, it uses 4-space
https://github.com/opendistro-for-elasticsearch/sql/blob/2939081a1927f8886cd9c16cb03430b6a6352c6e/ppl/src/main/java/com/amazon/opendistroforelasticsearch/sql/ppl/PPLService.java#L60